src/pages/EditProfilePage.js

```diff
@@ -256,7 +255,7 @@ const EditProfilePage = () => {
                 fullWidth
                 required
                 sx={editProfileClasses.nameField}
-                onChange={(e) => setEmail(e.target.value)}
+                onChange={(e) => user._id === userId && setEmail(e.target.value)}
```

Comment:
Create a variable that check if the `user._id === userId`  In my opinion it will be easier to read

Reply:
Yeah you are right.